src/pgp.rs

```diff
@@ -45,4 +45,33 @@ mod tests {
         )
         .await
     }
+
+    #[tokio::test]
+    // Make sure that parsing a .deb is ok
+    async fn test_verify_deb_error() -> Result<(), Error> {
+        let deb_path = "data/tests/fake_deb_for_unit_testing.none.deb";
+
+        let args: Args = Args::parse_from(&["app_name"]);
```

Comment:
`Args::parse_from` is tricky, as I need to import `clap:Parser` for it to work, but the compiler give me some warning.

I think I will instead refactor the code to avoid usage of `Args` in `pkg` & `async`. 



```
warning: unused import: `clap::Parser`
warning: unused import: `crate::args::Args`
warning: `spotify-launcher` (lib) generated 2 warnings (run `cargo fix --lib -p spotify-launcher` to apply 2 suggestions)

```

Reply:
Fixed via #[allow(unused_imports)]